akka-remote/src/test/scala/akka/remote/RemotingSpec.scala

```diff
@@ -637,7 +635,7 @@ class RemotingSpec extends AkkaSpec(RemotingSpec.cfg) with ImplicitSender with D
       val config = ConfigFactory.parseString(s"""
         akka.remote.enabled-transports = ["akka.remote.test"]
         akka.remote.retry-gate-closed-for = 5s
-        akka.remote.log-lifecylce-events = on
+        akka.remote.log-remote-lifecycle-events = on
```

Comment:
I am not sure if this is actually needed. Maybe comment it out, so it can be easily enabled?

Good catch btw.